style/styles.css

```diff
@@ -0,0 +1,1101 @@
+  width: 400px;
+  height: 100px;
+  background: white;
+  font-size: 35px;
+  color: #277dcc;
+  border-radius: 25px;
+}
+
+.smashingButton2 {
+  width: 300px;
+  height: 60px;
+  background: white;
+  font-size: 20px;
+  color: #277dcc;
+  border-radius: 25px;
+}
+
+.catSmash {
+  width: 245px;
+}
+
+.catTv {
+  height: 290px;
+}
+
+.smashingAd2 {
+  grid-column: span 2; /* with this the element can set a different width from the other elements */
+  display: block;
+  text-align: center;
+  border-radius: 25px;
+}
+
+.smashingContain2 {
+  background-color: #555;
+  height: 628px;
+  border-radius: 25px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  color: white;
+}
+
+.smashingFooter {
+  font-size: 12px;
+  font-weight: 300;
+}
+
+.textContainer {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin: 40px;
+}
+
+/* Articles Action Effects */
+.articlesDes:hover .redArrow {
+  background: #277dcc;
+}
+
+.articlesDes:hover .textContainer {
+  color: hsl(5, 65%, 51%);
+}
+
+.articlesDes:hover .redArrow .fa {
+  color: white;
+}
+
+.articlesDes:hover {
+  background: white;
+  transform: scale(1.05);
+}
+
+/* Membership Container */
+.wholeMembership {
+  background: #1f2c43;
+  justify-content: center;
+}
+
+.membership {
+  background: #1f2c43;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap-reverse;
+  justify-content: center;
+  align-items: center;
+  height: 500px;
+  color: white;
+  font-size: 22px;
+  width: 70%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.membershipText {
+  flex: 30%;
+}
+
+.membershipText p {
+  margin-bottom: 40px;
+}
+
+.membershipText ul {
+  margin-left: 60px;
+  margin-bottom: 40px;
+}
+
+.membershipImage {
+  width: 80%;
+  flex: 30%;
+  padding-left: 100px;
+  padding-bottom: 100px;
+  justify-content: center;
+  transform: rotate(-15deg);
+  transform-origin: left bottom;
+}
+
+.membershipImage img {
+  transform: scale(0.85);
+}
+
+.membershipImage:hover {
+  transform: rotate(0deg);
+}
+
+.blu a {
+  text-decoration: none;
+  color: white;
+}
+
+.footerMenuItems p {
+  width: 26%;
+}
+
+.footerMenuItems a {
+  font-size: 1.5rem;
+  color: #fff;
+  line-height: 2;
+  text-decoration: none;
+}
+
+.li-copyright a {
+  color: #fff;
+}
+
+.footerMenuItems p a {
+  color: #fff;
+  font-size: 1rem;
+  line-height: 2;
+}
+
+.membership p a {
+  color: #fff;
+}
+
+.linkTexts h2 a {
+  color: #666;
+}
+
+.linkTexts h3 a {
+  text-decoration: none;
+  color: #333;
+}
+
+.personOfWeek h4 {
+  font-size: 22px;
+  margin-bottom: 30px;
+}
+
+.personOfWeek h4 a {
+  color: #1374c8;
+}
+
+.footerMenuItems a:hover {
+  color: #333;
+}
+
+.membershipButton {
+  background-color: #4caf50; /* Green */
+  border: none;
+  color: white;
+  padding: 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 4px 2px;
+  cursor: pointer;
+  border-radius: 12px;
+  font-size: 100%;
+  text-shadow: #1f2c43;
```

Comment:
You have only added color to the `text-shadow`. But it needs two more values -> `h-shadow` and ` v-shadow`. 
https://www.w3schools.com/cssref/css3_pr_text-shadow.asp

Reply:
Changes implemented.